contracts/SpokePool.sol

```diff
@@ -25,6 +25,15 @@ interface AcrossMessageHandler {
         address relayer,
         bytes memory message
     ) external;
+
+    // New function interface to be used with USS functions since fillCompleted no longer has any
+    // meetings now that partial fills are impossible.
+    function handleUSSAcrossMessage(
```

Comment:
We should probably loop in Ryan to make sure we feel good about changing this interface on integrators. This means that the moment this change goes into effect, all integrators must have implemented the new function (even if its alongside the old function). If they haven't their relays will revert.